source/operator.rst

```diff
@@ -79,6 +79,10 @@ main subsystems of the cluster:
      - ``SmartUpdate``
      - A strategy the Operator uses for :ref:`upgrades<operator-update>`
 
+   * - autoPXCRecovery
+     - boolean
+     - ``true``
+
 .. _operator.upgradeoptions-section:
```

Comment:
seems we are missing `haproxy` section in the top of this page in the table

Reply:
I already see it in pxc-docs branch, perhaps it was brought there in different PR